diagnostic.md

```diff
@@ -35,21 +35,21 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+ This is a relative path because it has reference to the current directory on our computer.
```

Comment:
This would be an absolute. Reason being, you start at the root, `/Users/blah_blah/Desktop`. Anytime you start at the root for a path, it is an absolute path.